docs/Config_Reference.md

```diff
@@ -3166,6 +3166,9 @@ run_current:
 #   set, "stealthChop" mode will be enabled if the stepper motor
 #   velocity is below this value. The default is 0, which disables
 #   "stealthChop" mode.
+#driver_MULTISTEP_FILT: True
+#   Set this field to False if you get skipped steps in StealthChop2
+#   at high acceleration rates.
```

Comment:
Done.

> for example, on a tmc2208 changing multistep_filt has no effect because we use DEDGE mode.. It only impacts stealthchop on some drivers..

I just saw this in the datasheet. So it seems there are two cases of `multistep_filt`:
- tmc2208/tmc2209: They explicitly say not to use `dedge` and `multistep_filt` at the same time.
- tmc2240/tmc5160: They don't mention anything about `multistep_filt` and `dedge` having things in common.

But this only raises multiple questions from me:
- Why was `multistep_filt` enabled on tmc2208/tmc2209 by default in klipper? Don't all tmc drivers use `dedge` with klipper if supported by the MCU?
- Why was `multistep_filt` disabled on tmc5160 if there seems to be no link between `dedge` and `multistep_filt` on that driver? Was it simply forgotten here?